### PR TITLE
Add flag to enable/disable presence

### PR DESCRIPTION
**Pull request by Half-Shot**

### CODE REVIEW

File: lib/Main.js [hunk 1]
```diff
@@ -486,8 +486,8 @@ Main.prototype.getOrCreateBridgedRoom = function(gitterName) {
         main: this,
         gitter: this._gitter,
         gitterRealtime: this._gitterRealtime,
-
         gitterRoomName: gitterName,
+        enablePresence: !(this._config.enable_presence === false)
```
Comment (lukebarnard1) [inline lib/Main.js]: Why the `!(... === false)` and not just the thing?

Comment (Half-Shot) [reply lib/Main.js]: If missing we want it to be enabled by default. The only time we want it disabled is if it's explicitly set as false.

Comment (lukebarnard1) [reply lib/Main.js]: It's probably a better idea to move that logic to the BridgedRoom constructor. And doing an explicit = enablePresence === undefined ? true : enablePresence

Comment (Half-Shot) [reply lib/Main.js]: Fair point, will do.

**Comment by Half-Shot** [comment]
:tada:

**Pull request merged**